modules/offers/client/directives/tr-offer-host-view.client.directive.js

```diff
@@ -35,7 +35,7 @@
       vm.isUserPublic = false;
       vm.hostingDropdown = false;
       vm.hostingStatusLabel = hostingStatusLabel;
-
+      vm.isMobile = navigator.userAgent.toLowerCase().indexOf('mobile') >= 0;
```

Comment:
Let's access `navigator` via `$window.navigator` Angular service and add `$window` to the list on line 25. Not a biggie but it's just more Angular way of accessing globals.

While at it, let's also a check if `$window.isNativeMobileApp` is true. That's set by our native mobile app bridge (http://android.trustroots.org).